lib/memstore.ts

```diff
@@ -72,43 +72,25 @@ export class MemoryCookieStore extends Store {
     domain: string | null,
     path: string | null,
     key: string | undefined,
-  ): Promise<Cookie | null | undefined>
+  ): Promise<Cookie | undefined>
```

Comment:
There was only one path using `null`, so I changed it to `undefined` for consistency.